api/pkg/backend/service.go

```diff
@@ -703,15 +703,15 @@ func (s *APIService) UpdateTier(request *restful.Request, response *restful.Resp
 	log.Info("Check whether Tenant already exists before adding")
 	tenantExist := utils.ResourcesAlreadyExists(res.Tier.Tenants, updateTier.AddTenants)
 	if len(tenantExist) > 0 {
-		errMsg := fmt.Sprintf("some backends in AddBackends request: %v already exists in tier", tenantExist)
+		errMsg := fmt.Sprintf("some backends in AddBackends request: %v already exists in service plan", tenantExist)
 		log.Error(errMsg)
 		response.WriteError(http.StatusBadRequest, errors.New(errMsg))
 		return
 	}
 	log.Info("Check Tenant should exists before removing")
 	tenantNotExist := utils.ResourcesCheckBeforeRemove(res.Tier.Tenants, updateTier.DeleteTenants)
 	if len(tenantNotExist) > 0 {
-		errMsg := fmt.Sprintf("failed to update tier because delete backends: %v doesnt exist in tier", tenantNotExist)
+		errMsg := fmt.Sprintf("failed to update tier because delete backends: %v doesnt exist in service plan", tenantNotExist)
```

Comment:
failed to update tier ==> failed to update service plan

Reply:
changed